src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.Icu.cs

```diff
@@ -378,18 +378,15 @@ private static string ConvertIcuTimeFormatString(ReadOnlySpan<char> icuFormatStr
                         }
                         break;
 
-                    case ':':
-                    case '.':
                     case 'H':
                     case 'h':
                     case 'm':
                     case 's':
-                    case ' ':
-                    case '\u00A0': // no-break space
-                    case '\u202F': // narrow no-break space
                         result[resultPos++] = current;
                         break;
                     case 'a': // AM/PM
+                    case 'b': // am, pm, noon, midnight
+                    case 'B': // flexible day periods
```

Comment:
Should `B` be translated to `tt` instead? I didn't look at specs yet to see what correct translation should be.

Reply:
`b` is "am, pm, noon, midnight" and `B` is "flexible day periods," both of which should be acceptable to be translated to `tt` (AM/PM) as no other period is present in .NET if I'm not mistaken.

Reply:
Looking at: https://unicode-org.github.io/icu/userguide/format_parse/datetime/

|Symbol | Meaning | Pattern | Example Output
| - | - | - | -
a | AM or PM | a, aa, or aaaaaaaaaaaa | PM [abbrev]PM [wide]p
| b | am, pm, noon, midnight | b, bb, or bbbbbbbbbbbb | mid.midnightmd
| B | flexible day periods | B, BB, or BBBBBBBBBBBB | at night [abbrev]at night [wide]at night [narrow]

It looks to me `b` and `B` is not directly translated to `AM/PM`. We need to think more how we can support that in the .NET.

Reply:
True. But for now I think this is an acceptable solution.

Reply:
I suggest closing this PR till we figure out how we translate these symbols. Thanks @PopSlime for trying to help with that. Let's continue the discussion on the issue.

Reply:
> True. But for now I think this is an acceptable solution.

I am inclined to agree with that. This may be better than fully ignoring it.